### ISSUE TITLE
Error when using <objects><object>SOBJECT_NAME</object></objects> inside a <targetConfig>

### ISSUE THREAD

**Issue by AllanOricil** [forcedotcom/lightning-language-server#446]
### Summary

I just configured a LWC to work with an Object and I got this error. The configuration is fine and it is deployable to a Salesforce org. It is just the vscode extension that complains about it.

![image](https://user-images.githubusercontent.com/55927613/129906963-38d9a5c7-edb1-4fa4-ac5d-cf4400407744.png)

Below you can see the valid lwc xml config

````xml
<?xml version="1.0" encoding="UTF-8"?>
<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
    <apiVersion>52.0</apiVersion>
    <isExposed>true</isExposed>
    <targets>
        <target>lightning__RecordPage</target>
    </targets>
    <targetConfigs>
        <targetConfig targets="lightning__RecordPage">
            <property label="Height" name="height" type="String" default="500px" description="Adjust this value to change the Height. If you clean this value, it defaults to 100%."/>
            <property label="Width" name="width" type="String" default="" description="Adjust this value to change the component's Width. If you clean this value, it defaults to 100%."/>
            <objects>
                <object>WorkOrder</object>
            </objects>
        </targetConfig>
    </targetConfigs>
    
</LightningComponentBundle>
````

This is the error message when hovering over the indicated error:

````
Invalid element name:
 - objects

One of the following is expected:
 - property
 - propertyType
 - supportedFormFactors

Error indicated by:
 {http://soap.sforce.com/2006/04/metadata}
with code:xml(cvc-complex-type.2.4.a)
Limits the component to a set of one or more objects. If you don’t use this tag set, the component supports all supported objects. This tag set works only inside a parent targetConfig that’s configured for lightning__RecordPage. Specify the objects tag set only once inside a targetConfig set. Supports the object subtag
`````

### Steps To Reproduce:

1. Configure a "targetConfig" with an "Object", like the one below:
````xml
<targetConfigs>
        <targetConfig targets="lightning__RecordPage">
            <property label="Height" name="height" type="String" default="500px" description="Adjust this value to change the Height. If you clean this value, it defaults to 100%."/>
            <property label="Width" name="width" type="String" default="" description="Adjust this value to change the component's Width. If you clean this value, it defaults to 100%."/>
            <objects>
                <object>WorkOrder</object>
            </objects>
        </targetConfig>
    </targetConfigs>
````

### Expected result

This error message should not appear

### Actual result

There is a wrong error message appearing when configuring a LWC to work with an Object.

### Additional information

![image](https://user-images.githubusercontent.com/55927613/129907328-9a4352db-2d34-4647-a948-e937a67f6224.png)

**SFDX CLI Version**: sfdx-cli/7.94.3-a4e7c7955b win32-x64 node-v14.15.4

**OS and version**: windows 10

**Comment by randi274** [comment]
<img width="952" alt="Screen Shot 2022-03-07 at 5 19 53 PM" src="https://user-images.githubusercontent.com/9795193/157127622-64bff656-2349-446e-868c-0ba2b06f8f05.png">
Still seeing this as of March 2022, albeit slightly improved? Will log this as a bug for us, thanks!

**Comment by jeffb-sfdc** [comment]
This is a duplicate of https://github.com/forcedotcom/salesforcedx-vscode/issues/3503
